SU-SWS/stanford_paragraph_types
# Earth 302

# READY FOR REVIEW

# Summary
- Tall filmstrip paragraph type

# Needed By (Date)
- before you leave?

# Urgency
- low

# Steps to Test

1. checkout branch EARTH-302 in this and [stanford_components](https://github.com/SU-SWS/stanford_components/pull/65)
2. `drush en stanford_paragraph_tall_filmstrip -y`
3. create complex page with tall filmstrip component.
4. verify it looks close. Image needs some styles/refactoring to work with existing slide paragraph type. (don't think its necessary to change view modes just to change image style size)

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/65
https://github.com/SU-SWS/stanford_components/pull/67

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
